package.json

```diff
@@ -1,161 +1,9 @@
-    "prepack": "ember ts:precompile",
-    "postpack": "ember ts:clean"
-  },
-  "dependencies": {
-    "@ember/render-modifiers": "^2.0.0",
-    "@ember/test-waiters": "^3.0.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
-    "broccoli-funnel": "^3.0.8",
-    "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^5.7.1",
-    "ember-cli-typescript": "^4.2.1",
-    "ember-modifier": "^2.1.2 || ^3.0.0",
-    "tracked-built-ins": "^2.0.0"
-  },
-  "devDependencies": {
-    "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.4.2",
-    "@embroider/test-setup": "^1.0.0",
-    "@types/ember-qunit": "^3.4.15",
-    "@types/ember-resolver": "^5.0.10",
-    "@types/ember__application": "^3.16.3",
-    "@types/ember__array": "^3.16.4",
-    "@types/ember__component": "^3.16.6",
-    "@types/ember__controller": "^3.16.6",
-    "@types/ember__debug": "^3.16.5",
-    "@types/ember__destroyable": "^3.22.0",
-    "@types/ember__engine": "^3.16.3",
-    "@types/ember__error": "^3.16.1",
-    "@types/ember__object": "^3.12.6",
-    "@types/ember__polyfills": "^3.12.1",
-    "@types/ember__routing": "^3.16.15",
-    "@types/ember__runloop": "^3.16.3",
-    "@types/ember__service": "^3.16.1",
-    "@types/ember__string": "^3.16.3",
-    "@types/ember__template": "^3.16.1",
-    "@types/ember__test": "^3.16.1",
-    "@types/ember__test-helpers": "^2.6.0",
-    "@types/ember__utils": "^3.16.2",
-    "@types/htmlbars-inline-precompile": "^1.0.1",
-    "@types/qunit": "^2.11.2",
-    "@types/rsvp": "^4.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.6.0",
-    "@typescript-eslint/parser": "^5.6.0",
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "ember-auto-import": "^2.0.0",
-    "ember-cli": "3.28.0",
-    "ember-cli-content-security-policy": "^2.0.0-3",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-mirage": "^2.0.0",
-    "ember-cli-sass": "^10.0.0",
-    "ember-cli-terser": "^4.0.2",
-    "ember-cli-typescript-blueprints": "^3.0.0",
-    "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^1.0.0",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.4",
-    "ember-resolver": "^8.0.2",
-    "ember-source": "~3.28.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^4.0.0",
-    "ember-try": "^2.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.4",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.0.0",
-    "fake-xml-http-request": "^2.1.2",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.3.2",
-    "qunit": "^2.16.0",
-    "qunit-dom": "^2.0.0",
-    "release-it": "^14.0.0",
-    "release-it-lerna-changelog": "^4.0.0",
-    "sass": "^1.32.10",
-    "typescript": "^4.5.3",
-    "webpack": "5"
-  },
   "resolutions": {
```

Comment:
Currently `resolutions` are defined in `package.json` for workspaces root and in `package.json` for `ember-file-upload` package. Personally I find this difficult to reason about. I would recommend to merge them at one place for now.

Reply:
Oh yes you are right. I didn't pay attention. Great catch, will move those to workspace `package.json`.